### ISSUE TITLE
Instance: The `.chat-reply-button` controls have non-descriptive accessible names | WCAG 2.2 Violation - There are controls with non-descriptive accessible names present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#49]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- 

### User Impact

Some users - such as people using voice control software - may be unable to access, focus, or activate the controls. Screen reader users may be confused by the controls being announced by a different accessible name than their visible labels.

#### Further information:

1. 

### Screenshot

![Screenshot of an example of one of the \`.chat-reply-button\` controls (showing an accessible name of 'Reply to User 1 on message \'Test comment\').](https://github.com/user-attachments/assets/aba4cc0b-640c-493b-ae24-815c7939412a)

### Code Reference

```
<div id="comment-[…]" class="chat-message " data-comment-id="[…]">
	<div class="chat-message-content">
		[…]
		<div class="chat-bubble">
			[…]
			<div class="chat-actions">
				<button class="chat-reply-button" data-comment-id="[…]" data-author="[…]" aria-label="Reply to User 1 on message 'Test comment'">
					<svg aria-hidden="true" width="14" height="14" viewBox="0 0 16 16" fill="currentColor">[…]</svg>
					Reply
				</button>
			</div>
		</div>
	</div>
</div>
```
